### ISSUE TITLE
PDF Format not there

### ISSUE THREAD

**Issue by patrysia181** [waylaidwanderer/FicSave#120]
So I just found myself wanting to download a story into a PDF format but I noticed it's not there anymore... Anyone know what's going on?

**Comment by waylaidwanderer** [comment]
https://twitter.com/FicSave

If you read the feed you'll see why. In short the server kept running out of memory due to PDF conversions and caused more trouble than it was worth. I'm considering alternatives in the meantime.

**Comment by KitKat71483** [comment]
Oh no! I was hoping it was just a short term thing with not having PDF available.  Are you considering making a way for PDF to be used again at a later date?

**Comment by waylaidwanderer** [comment]
Yes, I'm trying to find other solutions for PDF creation, so hopefully it won't be gone forever.

**Comment by timmbarney94** [comment]
Could you use off instead of the others or does the off take up the most space of the other 3
